### DISCUSSION TITLE
Livewire V3 databinding issue

### DISCUSSION THREAD

**Discussion by Gopal-Bandyopadhyay** [livewire/livewire#6116] [7. Bugs (archived)]
### Livewire version

v3

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

Like the Livewire V2 data binding example where any changes in message variable in text input reflect on output directly but in the Livewire V3 data binding example we need a call a function to reassign the variable to get updated value. Same example available in Livewire V2 not working in V3 or Variable not updating automatically when typing on input box.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@Gopal-Bandyopadhyay thanks for reporting! Any chance you can provide some code that demonstrates the issues so we can investigate? Thanks!

**Reply by techinfocomp** [reply]
Almost the same issue.
https://github.com/livewire/livewire/discussions/6644

1st try to increase and decrease the value by + and – button.
Then just update the input text value with any number and try again to press the + and – button.
And see the result.

**Reply by mdbilal** [reply]
or you can simply use `wire:model.live` instead of just `wire:model`

**Comment by mdbilal** [comment]
> ### Livewire version
> v3
> 
> ### Which PHP version are you using?
> PHP 8.2
> 
> ### Repository URL
> _No response_
> 
> ### Steps To Reproduce
> Like the Livewire V2 data binding example where any changes in message variable in text input reflect on output directly but in the Livewire V3 data binding example we need a call a function to reassign the variable to get updated value. Same example available in Livewire V2 not working in V3 or Variable not updating automatically when typing on input box.
> 
> ### Additional information
> _No response_
> 
> ### Please confirm (incomplete submissions will not be addressed)
> * [x]  I have provided easy and step-by-step instructions to reproduce the bug.
> * [x]  I have provided code samples as text and NOT images.
> * [x]  I understand my bug report will be removed if I haven't met the criteria above.

The simple answer to your question is you should use `wire:model.live` instead of just `wire:model`

Here's the explanation from the Livewire documentation:

"Why isn't my component live updating as I type?"
If you tried this in your browser and are confused why the title isn't automatically updating, it's because Livewire only updates a component when an "action" is submitted—like pressing a submit button—not when a user types into a field. This cuts down on network requests and improves performance. To enable "live" updating as a user types, you can use wire:model.live instead.
